torch/backends/cuda/__init__.py

```diff
@@ -317,6 +318,15 @@ def sdp_kernel(
     This context manager can be used to temporarily enable or disable any of the three backends for scaled dot product attention.
     Upon exiting the context manager, the previous state of the flags will be restored.
     """
+    warnings.warn(
+        (
+            "torch.backends.cuda.sdp_kernel() "
```

Comment:
Can we rewrite it in terms of sdpa_kernel? Then it could also serve as an example.

Reply:
"will be removed in 2.6"?

Reply:
I updated this to write in terms of sdpa_kernel